examples/workflow.yaml

```diff
@@ -68,7 +77,7 @@ steps:
       module_name: neal
       function_name: SimulatedAnnealingSampler
     type: specs
-  - sample_params:
+  - sampler_params:
```

Comment:
there more parameters that could be used here. 
nsweeps, nreads, \beta (i.e. inverse temeprature)
Why dont we pass these in the example?

Reply:
Because I didn't feel like adding them ;)
But that's actually a good point to have them in the example, will add.

Reply:
Fixed here: https://github.com/zapatacomputing/z-quantum-qubo/pull/7/commits/67196b3d45d2468a1f7227d169c9bae10f77d408

Reply:
I think Orquestra will support default values after Monday. They deleted the verification layer thing?

Reply:
So Orquestra now requires to specify "input artifacts". But if your input artifact is a dict, then you can extend this dict as much as you want and Orquestra won't complain.

As a sidenote: I wouldn't be sure about Monday, but I might be missing some info.